src/Configuration/Config.AzureKeyVault/src/AzureKeyVaultConfigurationSource.cs

```diff
@@ -25,6 +25,16 @@ internal class AzureKeyVaultConfigurationSource : IConfigurationSource
         /// </summary>
         public IKeyVaultSecretManager Manager { get; set; }
 
+        /// <summary>
+        /// Determines whether the source will be loaded if the Azure KeyVault changes.
+        /// </summary>
+        public bool ReloadOnChange { get; set; }
+
+        /// <summary>
+        /// Number of milliseconds to wait inbetween each attempt at polling the Azure KeyVault for changes.
+        /// </summary>
+        public int ReloadPollDelay { get; set; } = 1000;
```

Comment:
Polling interval can't be negative - let's change this to an uint type.

Reply:
Followed the same pattern with the poll delay for the file extension configuration source, though i agree that it should be a uint.